torch/ao/quantization/fx/_model_report/model_report.py

```diff
@@ -415,3 +419,21 @@ def generate_visualizer(self) -> ModelReportVisualizer:
         visualizer: ModelReportVisualizer = ModelReportVisualizer(module_fqns_to_features)
 
         return visualizer
+
+    def generate_qconfigs(self) -> Dict[str, QConfigMapping]:
```

Comment:
may be better to have an equalization generator and a qconfigmapping generator since this type of dict object of qconfigmappings isn't something that exists elsewhere.

Reply:
would be better to name this either generate_qconfig_mappings or generate_qconfig_mapping/generate_equalization_config i.e. the actual name of the objects, because a qconfig is something different from a qconfig mapping